feat: Add closed contribution process (#316)
nginx/documentation: F5-NGINX-team-notes.md

# For F5/NGINX Employees

> This repository is a functional mirror. We want to do as much of our work as possible in the public repository. 

Author: ```suggestion
This repository is public, and do as much of our work as possible in the public as a commitment to open source.
```

> Content which cannot be publicised before release is handled slightly differently: please read the [Contributing guidelines for closed content](/CLOSED_CONTRIBUTIONS.md) document.

Reviewer: ```suggestion
If you're working with content which cannot be publicized before release is handled slightly differently: please read the [Contributing guidelines for closed content](/CLOSED_CONTRIBUTIONS.md) document.
```

Author: I need to re-read this in context but I think I specifically avoided the phrasing of "If you're working" due to repetition.

Reviewer: I think "active voice" is important here

> Before new content is published at https://docs.nginx.com, it must be commited to this `documentation` repository. 

Reviewer: ```suggestion
Before new content is published at https://docs.nginx.com, it must be committed to this `documentation` repository. 
```

# For F5/NGINX Employees

This repository is public, and we do as much of our work as possible in the public as a commitment to open source.

Before new content is published at https://docs.nginx.com, it must be committed to this `documentation` repository. 

If you are an F5 employee unable to assign yourself as an owner of an issue or a reviewer of a pull request, ask a member of the NGINX documentation team for help.

> If your content cannot be publicised before release, it is handled differently: read the [Contributing guidelines for closed content](/CLOSED_CONTRIBUTIONS.md) document.

Reviewer: American English...

```suggestion
If your content cannot be publicized before release, it is handled differently: read the [Contributing guidelines for closed content](/CLOSED_CONTRIBUTIONS.md) document.
```
